ambry-store/src/main/java/com.github.ambry.store/StoreMetrics.java

```diff
@@ -25,36 +25,39 @@
  * Metrics for a specific store.
  */
 public class StoreMetrics {
```

Comment:
+1 lets not change the metrics classes just yet. Intellij hinting is good when you are writing new code and the new code you write is highlighted. Even then, you should take a call on what needs fixing and what needs to remain as is.